workspaces/ui-v2/src/spectacle-implementations/cloud-viewer.tsx

```diff
@@ -122,11 +128,19 @@ export default function CloudViewer() {
               >
                 <SpecMetadataProvider>
                   <Switch>
-                    <>
-                      <DiffReviewEnvironments />
-                      <DocumentationPages />
-                      <ChangelogPages />
-                    </>
+                    <Route
+                      path={`${match.path}/changes-since/:batchId`}
```

Comment:
We can't use the `navigation/Routes` helpers since it relies on baseUrl - which is only rendered in this component. We could move this out into a separate component. 

Side note, I'm generally not a fan of the current route helpers - it provides a centralized way to specify routes but there's a fair amount of indirection we need to go through (hook -> function call / path param) - but we still have somewhat scattered route logic (i.e. useParams needs to know what context it's rendered in, and what the named path params are)